.golangci.yml

```
@@ -0,0 +1,66 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
+
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - typecheck
+    - dupl
+    - goprintffuncname
+    - govet
+    - nolintlint
+    - rowserrcheck
+    #- gofmt
```

```suggestion
    - gofmt
```
Should be good as we use gofmt already

---

no, this file is not use gofmt.

pkg/scalers/azure/azure_monitor_test.go:19: File is not `gofmt`-ed with `-s` (gofmt)

---

That's interesting as `make gofmt` finds no issues

---

Maybe  gofmt and go  fmt is different

---

We use `go fmt` 😄 
https://github.com/kedacore/keda/blob/3a99b77cc330068a139333cca89b30aa1145a375/Makefile#L199-L202

---

I saw from the introduction that go fmt is just a simple encapsulation of gofmt, essentially using gofmt. The reason for the different performance, I guess it may be that the referenced gofmt version may be different (or other reasons).  But the effect is not big, and I think it might be more suitable to use gofmt directly